table/server/underdb/glue/src/main/java/alluxio/table/under/glue/GlueTable.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.table.under.glue;
+
+import alluxio.grpc.table.ColumnStatisticsInfo;
+import alluxio.grpc.table.FieldSchema;
+import alluxio.grpc.table.Layout;
+import alluxio.grpc.table.Schema;
+import alluxio.grpc.table.layout.hive.PartitionInfo;
+import alluxio.table.common.UdbPartition;
+import alluxio.table.common.layout.GlueLayout;
+import alluxio.table.common.udb.UdbTable;
+import alluxio.table.under.glue.util.PathTranslator;
+
+import com.amazonaws.services.glue.model.Column;
+import com.amazonaws.services.glue.model.EntityNotFoundException;
+import com.amazonaws.services.glue.model.Partition;
+import com.amazonaws.services.glue.model.Table;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Glue table implementation.
+ */
+public class GlueTable implements UdbTable {
```

Comment:
what is the difference between this class and the HiveTable class? can we refactor to reduce duplicated code?

Reply:
There is no difference for the current version, should we just use the hive layout instead?